packages/sdk/src/create/libp2p.ts

```diff
@@ -106,8 +106,13 @@ export async function createLibp2pAndUpdateOptions(
     peerDiscovery.push(...getPeerDiscoveries(pubsubTopics, options.discovery));
   }
 
-  if (options?.bootstrapPeers) {
-    peerDiscovery.push(bootstrap({ list: options.bootstrapPeers }));
+  const bootstrapPeers = [
+    ...(options.bootstrapPeers || []),
+    ...(options.store?.peers || [])
```

Comment:
nit: general comment on naming, I know from context this is supposed to be peer store and not store protocol but it might cause confusion